tempesta_fw/http.c

```diff
@@ -759,6 +759,8 @@ tfw_http_adjust_resp(TfwHttpResp *resp, TfwHttpReq *req)
 	if (r < 0)
 		return r;
 
+	tfw_http_msg_set_conn(hm, conn_flg);
```

Comment:
Nice touch, considering that flags **are** checked now in a response on the way out.